plugin/helpers.go

```diff
@@ -14,7 +14,9 @@ type Helpers interface {
 
 	// KVSetJSON stores a key-value pair, unique per plugin, marshalling the given value as a JSON string.
 	//
-	// Minimum server version: 5.2
+	// Minimum server version: 5.18
```

Comment:
do we need to modify the version here? Isn't just adding ` Deprecated: Use p.API.KVSetWithOptions instead.` enough?

Reply:
I've reverted the changes.